tools/cldr-apps/js/src/esm/cldrDeferHelp.mjs

```diff
@@ -110,7 +96,29 @@ WHERE {
  */
 function sparqlQuery(query, endpoint) {
   endpoint = endpoint || defaultEndpoint;
-  return $.getJSON(endpoint, { query, format });
+  const url = new URL(endpoint);
+  url.searchParams.append("query", query);
+  url.searchParams.append("format", "JSON");
+  return fetch(url);
```

Comment:
I like to use `cldrAjax.doFetch` but I guess it's not appropriate for a non-unicode.org server. We might consider a new method cldrAjax for other servers. It's good to go thru a single module for http.

Reply:
That would leak the session key! I would be fine with a generic fetch, it could even just be re-exported fetch() would that work